src/db.ts

```diff
@@ -302,25 +302,25 @@ export const saveBooking = (accountId, booking) => {
     .then(savePerson);
 };
 
-export const getAllPersons = (): Promise<MockPerson[]> => {
-  return redisClient
-    .keysAsync(`${process.env.MOCKSOLARIS_REDIS_PREFIX}:person:*`)
-    .then((keys) => {
-      if (keys.length < 1) {
-        return [];
-      }
-      return redisClient.mgetAsync(keys);
-    })
-    .then((values) => values.map(jsonToPerson))
-    .then((values) =>
-      values.sort((p1, p2) => {
+export const getAllPersons = async (
+  sort: boolean = false
+): Promise<MockPerson[]> => {
+  const keys = await redisClient.keysAsync(
+    `${process.env.MOCKSOLARIS_REDIS_PREFIX}:person:*`
+  );
+  if (keys.length < 1) return [];
+  const values = await redisClient.mgetAsync(keys);
+  let persons = values.map(jsonToPerson);
+  persons = sort
+    ? persons.sort((p1, p2) => {
         if (!p1.createdAt && p2.createdAt) return 1;
         if (p1.createdAt && !p2.createdAt) return -1;
         if (!p1.createdAt && !p2.createdAt) return 0;
         return p1.createdAt > p2.createdAt ? -1 : 1;
       })
-    )
-    .then((results) => results.map((person) => augmentPerson(person)));
+    : persons;
+  persons = persons.map((person) => augmentPerson(person));
```

Comment:
we could directly return here without new assigning

Reply:
Yes: https://github.com/kontist/mock-solaris/pull/112/commits/19ce037f552e64969cd24e3271c0b74e8b13a10e